### PR TITLE
cargo overalls

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

add title overalls in the game

## Why / Balance

drip or drown

## Technical details

no

## Media

Screenshot_20240915_173508

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added cargo overalls.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
"more clothing 'n drip" is not inherent justification to add something. Please actually state why this is a good idea to add, and to Cargo specifically. 
You also can't just write "no" in the technical details section.

File: Resources/Textures/Clothing/Uniforms/Jumpsuit/overalls_cargo.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at commit https://github.com/tgstation/tgstation/commit/ac792e3226d48e5b3aaccff165ce100f7636a040, monkey made by brainfood1183 (github) for ss14, rehued by lzk228 (github)",
```
Comment (Iamgoofball) [inline Resources/Textures/Clothing/Uniforms/Jumpsuit/overalls_cargo.rsi/meta.json:4]: You appear to have copypasted this as Swept wasn't involved in these sprites at all. Also, this is a commit for the mailman jumpsuit.

Comment (lzk228) [reply Resources/Textures/Clothing/Uniforms/Jumpsuit/overalls_cargo.rsi/meta.json:4]: i copied that from botany overalls copyright and added rehued part, i suppose Swept was involved in botany overalls sprite

**Comment by Emisse** [comment]
needs art maintainer approval

**Comment by UbaserB** [comment]
Clothing is frozen as per issue no. 8524, so I'm closing this PR. If the freeze ever gets undone, please ping me to reopen the PR.

**Pull request closed without merging**